admins/pageflow/accounts.rb

```diff
@@ -139,5 +139,22 @@ def permitted_attributes_for(resource_name)
         end
       end
     end
+
+    member_action 'confirm_invitations',
+                  title: 'pageflow.admin.users.confirm_invitations', method: :get do
+      account = Pageflow::Account.find(params[:id])
+
+      invitations = account.invitations.by_user(current_user)
+
+      if invitations.empty?
+        redirect_to admin_root_path,
+                    alert: I18n.t('pageflow.admin.users.no_invitations_available')
+      else
+        invitations.turn_into_memberships
+
+        redirect_to admin_root_path,
+                    notice: I18n.t('pageflow.admin.users.invitations_confirmed')
+      end
+    end
```

Comment:
Let's move this method to the invitations admin. There is already too much going on here. The route would have been nice, but `/admin/invitations/confirm?account_id=5` works too.